.husky/custom/block-master.sh

```diff
@@ -0,0 +1,7 @@
+branch="$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
This is husky precommit, making sure that if we are on branch master then it does not allow to commit